usr/share/rear/prep/DP/default/450_check_dp_client_configured.sh

```diff
@@ -1,15 +1,18 @@
 # 450_check_dp_client_configured.sh
 # this script has a simple goal: check if this client system has been properly
-# defined on the DP cell server and that a backup specification has been
-# made for this client - no more no less
+# configured on the Cell Manager and backed up at least once - no more no less
 
 OMNIDB=/opt/omni/bin/omnidb
 OMNIR=/opt/omni/bin/omnir
 
 Log "Backup method is DP: check Data Protector requirements"
+[ -x ${VBDA} ]
+StopIfError "Please install Data Protector Disk Agent (DA component) on the client."
+
 [ -x ${OMNIR} ]
-StopIfError "Please install Data Protector User Interface (cc component) on the client."
+StopIfError "Please install Data Protector User Interface (CC component) on the client."
 
 ${OMNIDB} -filesystem | grep $(hostname) >/dev/null
```

Comment:
I wonder if
```
${OMNIDB} -filesystem | grep -q $(hostname)
```
woudn't be the more direct and straightforward way
than STDOUT redirection to /dev/null?